pkgs/development/python-modules/sphinxcontrib-fulltoc/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, sphinx, pbr }:
+
+buildPythonPackage rec {
+
+  pname = "sphinxcontrib-fulltoc";
+  version = "1.2.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1nbwflv9szyh37yr075xhck8b4gg2c7g3sa38mfi7wv7qhpxcif8";
+
+  };
+
+  buildInputs = [ pbr ];
+  propagatedBuildInputs = [ sphinx ];
+
+  # There are no unit tests
+  doCheck = false;
+
+  meta = {
```

personal preference, then remove lib references below please :)
```suggestion
  meta = with lib; {
```